.travis.yml

```diff
@@ -0,0 +1,82 @@
+---
+
+###
+### Travis settings
+###
+sudo: required
+services:
+  - docker
+
+
+###
+### Env variables
+###
+env:
+  global:
+    - image=flaconi/java-base
+
+jobs:
+  include:
+    ###
+    ### Stage 1: Static file analysis
+    ###
+    - stage: static file analysis
+      install:
+        - max=100; i=0; while [ $i -lt $max ]; do if sudo apt-get update; then break; else i=$((i+1)); fi done
+        - max=100; i=0; while [ $i -lt $max ]; do if sudo apt-get -y install moreutils; then break; else i=$((i+1)); fi done
+        - git clone https://github.com/cytopia/awesome-ci /tmp/awesome-ci
```

Comment:
I'm not sure we actually need this (yet). What about introducing [Anchore](https://anchore.io) at this point for? Besides, shouldn't we fork your repo first under the company's GH account?